forloop_modules/queries/db_model_templates.py

```diff
@@ -1,11 +1,13 @@
+import datetime
+from enum import Enum
+from typing import Any, Dict, Generic, List, Optional, TypeVar, Union
+
 from pydantic import BaseModel, Field
 from pydantic.functional_validators import field_validator
-from typing import Optional, Any, List, Dict, Union, Generic, TypeVar
-import datetime
-import re
 
-from enum import Enum
-    
+from forloop_common_structures.core.trigger import TriggerFrequencyEnum
+from forloop_common_structures.core.database import DbDialectEnum
```

Comment:
put in Forloop Modules 
![obrazek](https://github.com/ForloopAI/forloop_modules/assets/29150831/c39a22fb-db97-4310-9f7c-0d36b70c7957)